apps/system/style/notifications/notifications.css

```diff
@@ -146,17 +147,19 @@
 
 #notification-bar button {
   flex: initial;
-  width: auto;
+  width: 9.5rem;
   height: 4.5rem;
-  padding: 0;
+  padding: 0 0 0 1.5rem;
   border: 0;
-  background: none;
-  color: #008eab;
-  font: italic normal 1.3rem/1.3rem auto;
+  border-radius: 0;
+  color: #000000;
+  background-color: #00caf2;
+  font: italic normal 2.3rem/2.3rem auto;
 }
 
 #notification-bar button[disabled] {
   color: rgba(255, 255, 255, 0.2);
+  background-color: transparent;
```

Comment:
this change won't be required once the background change is only in the `:active` case